src/res/other_data.inc

```diff
@@ -607,10 +606,10 @@ sTGM3SpeedCurveEnd::
 
 sTGM3SpeedCurveSpecialData::
     dw $FFFF    ; When do we get bones?
-    dw 1299     ; When do blocks turn invisible?
```

Comment:
I haven't seen any code that skips sections, is it okay to shorten the curve?

Reply:
until we find a way to skip sections, yes